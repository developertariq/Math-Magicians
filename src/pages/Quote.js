import React from 'react';

const Quote = () => (
  <div className="main-container">
    <h2
      className="page-title"
      style={{
        padding: '2.4em',
        marginTop: '3rem',
        lineHeight: '1.45em',
      }}
    >
      Mathematics is not about numbers, equations, computetions, or algorithms.
      It is about understanding -
      <span style={{ fontStyle: 'italic' }}>
        William Paul Thurston
      </span>
    </h2>
  </div>
);

export default Quote;
