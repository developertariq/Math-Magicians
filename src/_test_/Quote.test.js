import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Quote from '../pages/Quote';

describe('Quote', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
