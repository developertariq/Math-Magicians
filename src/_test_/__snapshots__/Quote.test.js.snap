// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
<div
  className="main-container"
>
  <h2
    className="page-title"
    style={
      Object {
        "lineHeight": "1.45em",
        "marginTop": "3rem",
        "padding": "2.4em",
      }
    }
  >
    Mathematics is not about numbers, equations, computetions, or algorithms. It is about understanding -
    <span
      style={
        Object {
          "fontStyle": "italic",
        }
      }
    >
      William Paul Thurston
    </span>
  </h2>
</div>
`;
